### ISSUE TITLE
Challenges - Submenu

### ISSUE THREAD

**Issue by D3Portillo** [Arena-Mayhem/frontend#12]
## TL;DR

Allow users to see "active" games and all previous games + results (history).

## Component Story

- As a user I want to see an "ONGOING" button
- As a user I want to see a "HISTORY" button
- I want to see an "active" state that: shows the description, a custom border and a gradient background
- I want to have an "inactive" state that hides the description, custom border and gradient
- We want to save the user selection (maybe using a route like /challenges, challenges/history or using a state library)
- When a user is in a mobile device the selection is shown in a list
- When a user is in Desktop device the selection of buttons is shown in columns

## References

![Image](https://github.com/user-attachments/assets/e8a96ef3-160d-46e7-8538-b898c9e83620)

### Mobile

![Image](https://github.com/user-attachments/assets/a1d482fe-86ad-4cc0-ad29-7f286476d800)

## Resources

- Figma View: https://www.figma.com/design/csb1dd9yjNAg4sWTmVEZww/Arena-Mayhem?node-id=770-17744&t=htvhpzMvSyVVJfRj-0

---

> [!TIP]
> You can use shadcn's UI to create this view. An example using Tabs Component: https://ui.shadcn.com/docs/components/tabs

**Comment by cyanidev** [comment]
![Screenshot_11](https://github.com/user-attachments/assets/25f818c1-a727-4842-a06b-f85000530851)
![Screenshot_10](https://github.com/user-attachments/assets/b963d05e-3679-46d3-873e-6068603514d4)
